publish_data.go

```diff
@@ -130,22 +155,55 @@ func (dp DataPublisher) PublishData(records []*DataRecord) error {
 	return nil
 }
 
-func packet(rec *DataRecord) ([]byte, []byte) {
-	// Structure of the message header is defined in BINARY_FORMATS.md
-	// 16 bits: channel number
-	//  8 bits: header version number
-	//  8 bits: code for data type (0-1 = 8 bits; 2-3 = 16; 4-5 = 32; 6-7 = 64; odd=uint; even=int)
-	// 32 bits: # of pre-trigger samples
-	// 32 bits: # of samples, total
-	// 32 bits: sample period, in seconds (float)
-	// 32 bits: volts per arb conversion (float)
-	// 64 bits: trigger time, in ns since epoch 1970
-	// 64 bits: trigger frame #
+// messageSummaries makes a message with the following format for publishing on portTrigs
+// Structure of the message header is defined in BINARY_FORMATS.md
+// 16 bits: channel number
+//  8 bits: header version number
+//  8 bits: Presamples
```

Comment:
The number of bits listed in these comments, and the list of comments, is inconsistent with the `binary.Write()` calls below.